packages/@dcl/inspector/src/components/EntityInspector/TriggerInspector/TriggerInspector.tsx

```diff
@@ -71,17 +71,14 @@ export default withSdk<Props>(
     const hasCounter = useHasComponent(entityId, Counter)
 
     const areValidActions = useCallback(
-      (updatedActions: TriggerAction[]) =>
-        updatedActions.length > 0 && updatedActions.every((action) => action.id && action.name),
-      []
+      (updatedActions: TriggerAction[]) => updatedActions.every((action) => action.id && action.name),
+      [triggers]
     )
 
     const areValidConditions = useCallback(
-      (updatedConditions: TriggerCondition[] | undefined) =>
-        updatedConditions && updatedConditions.length > 0
-          ? updatedConditions.every((condition) => condition.id && condition.value)
-          : true,
-      []
+      (updatedConditions: TriggerCondition[] = []) =>
+        updatedConditions.every((condition) => condition.id && condition.value),
+      [triggers]
```

Comment:
why is it necessary to add `triggers` as a dependency if it's not used in the scope of the callbacks?

Reply:
good catch, was testing something and left this out, removed!